### PR TITLE
add RemoteClusterValues to integ test framework

### PR DESCRIPTION
**Pull request by GregHanson**
add RemoteClusterValues to distinguish between remote and non-remote values during multicluster install

related to https://github.com/istio/istio/pull/24373#issuecomment-640826601

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ X ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by GregHanson** [comment]
@howardjohn @stevenctl, wondering if in the case of RemoteClusterValues being empty, should ControlPlaneValues be used in its place?  For instance, in the existing multimaster and remote tests I had to set RemoteClusterValues and ControlPlaneValues to the same string.  If another multicluster test is written and this step missed, the remotes will most likely not install properly

File: tests/integration/multicluster/remote/main_test.go [hunk 1]
```diff
@@ -55,6 +55,7 @@ func TestMain(m *testing.M) {
 		SetupOnEnv(environment.Kube, istio.Setup(&ist, func(cfg *istio.Config) {
 			// Set the control plane values on the config.
 			cfg.ControlPlaneValues = controlPlaneValues
+			cfg.RemoteClusterValues = controlPlaneValues
```
Comment (howardjohn) [inline tests/integration/multicluster/remote/main_test.go:58]: this would have defaulted to ControlPlaneValues if you didn't set this right?

Comment (GregHanson) [reply tests/integration/multicluster/remote/main_test.go:58]: was wondering what the default behavior should be in this case.  Currently RemoteClusterValues will not default to ControlPlaneValues if not specified

Comment (stevenctl) [reply tests/integration/multicluster/remote/main_test.go:58]: What about cases where something should only go to control plane clusters and setting those values for a remote cluster breaks things, but you don't have any values you need to set on the remote clusters. Seems like an edge case but you'd have to set some dummy value for it to be ignored. Explicit seems better to me.

Comment (howardjohn) [reply tests/integration/multicluster/remote/main_test.go:58]: My main concern is now every test needs to explicitly declare remote control plane stuff. Don't we want to run all of our tests as multicluster?
